CHANGELOG.md

```diff
@@ -17,8 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `SubspaceDiscrete.to_searchspace` and `SubspaceContinuous.to_searchspace`
   convenience constructor
 - Validators for `Campaign` attributes
-_ `_optional` subpackage for managing optional dependencies
+- `_optional` subpackage for managing optional dependencies
 - Acquisition function for active learning: `qNIPV`
+- User guide for active learning
```

Comment:
Is the addition of an additional ACQF worth mentioning?

Reply:
it is mentioned above
its for an entirely new use case so its certainly worth mentioning

Reply:
sorry I just saw that I added a new acqf also inthis PR :D I guess you mean the `PosteriorStandardDeviation` ? that make sense to add it too agree

Reply:
added